### DISCUSSION TITLE
Existing transactions are re-added (duplicates) when running GoCardless after updating ff3.

### DISCUSSION THREAD

**Discussion by dekiesel** [firefly-iii/firefly-iii#8516] [Firefly III] [answered]
I just updated ff3 from `6.0.23` to `6.1.7` and the data importer from `1.3.8` to `1.4.3`.

I then used the data importer with the time range option (30 days) to get past transactions from GoCardless. I noticed that the button now say GoCardless. I had all the transactions that I got from GoCardless already in my ff3 DB, but the data importer added every one of them as new.

To update I simply ran `docker compose up -d`. Should I not have done that?

I have a backup, so no harm done, but I am now wondering how to update. If anybody has an idea please let me know.

**Comment by JC5** [comment]
Version 1.4.0 of the data importer probably caused this:

https://github.com/firefly-iii/data-importer/releases/tag/v1.4.0

**Reply by dekiesel** [reply]
Thanks. So the solution is to update but to not use the updater on transactions older than the update-date?

**Reply by JC5** [reply, marked as answer]
For example, yes. Personally, I limit the date range to 3 days, and run an import every day. That way, at least the amount of duplicates is limited.
